### ISSUE TITLE
geom boxplot doesn't require x aes!

### ISSUE THREAD

**Issue by dannyparsons** [IDEMSInternational/R-Instat#4807]
A very minor changed in ggplot2 v3.0.0 is that this now works:
`ggplot(mtcars, aes(y = disp)) + geom_boxplot()`
So x isn't required. This could simplify some of our code, although I don't like that the graph shows a scale for the x axis which is confusing as it doesn't really have any meaning:
![image](https://user-images.githubusercontent.com/11214914/43075324-5e9960f2-8e78-11e8-9663-8607ade120ef.png)
Compared to `ggplot(mtcars, aes(x = "", y = disp)) + geom_boxplot()` which is simpler.
![image](https://user-images.githubusercontent.com/11214914/43075349-74a8f79a-8e78-11e8-927e-6a17eb3dfc6a.png)

I'm not sure we need to change anything but it's good to know x is now optional. This is much better for teaching ggplot2 as it wasn't obvious to explain why x is needed for boxplots.

**Comment by maxwellfundi** [comment]
i agree this looks simpler. Possibly there could be a way to get rid of the scales when we have no x

**Comment by dannyparsons** [comment]
I've asked about this on the original ggplot2 issue so will see what the response is to that.

**Comment by maxwellfundi** [comment]
Great!
Regards
Maxwell Fundi
|| Phone :+254728508241||

**Comment by Ogik99** [comment]
@maxwellfundi, @dannyparsons  any updates on this

**Comment by dannyparsons** [comment]
The issue on ggplot2's repo was closed so I'm not sure if anything was done. Someone could check with the latest version of ggplot2 what happens with `ggplot(mtcars, aes(y = disp)) + geom_boxplot()` now.

**Comment by Ogik99** [comment]
The scale on the x axis still exists.
